data/strings/strings.txt

```diff
@@ -25171,6 +25227,7 @@ fa = دوباره نپرس
     el = Γενικό σχόλιο
     sk = Všeobecné pripomienky
     sw = Maoni Jumla
+    fa =  "تنظیمات: دکمه "ارسال بازخورد کلی"   
```

Comment:
I don't think the quotes are correct here.

Reply:
fixed